Dockerfile

```diff
@@ -57,6 +57,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 WORKDIR /app
 COPY ./bin/docker_start.sh /start.sh
 COPY ./bin/reset_migrations.sh /app/bin/reset_migrations.sh
+COPY ./bin/uninstall_adfs.sh ./bin/uninstall_django_auth_adfs_db.sql /app/bin/
```

Comment:
Should this already be included in the docker image for 1.7.0, if it should only be used for version 1.8.x and onward?

Reply:
I wanted to add it so we don't forget about it for 1.8 - so now it's basically undocumented.

If it's executed by accident, then migrations can run again and will re-create the tables (I presume), so running the script again then should be sufficient.

Reply:
Fair enough, it won't do any harm I think :+1: